HetDepot/Controllers/Tests/KevinsTestController.cs

```diff
@@ -19,34 +19,65 @@ class KevinsTestController : Controller
 
     public KevinsTestController()
     {
-        _toursPath = Path.Combine(Directory.GetCurrentDirectory(), "ExampleFile\\ExampleTours.json");
+        _toursPath = Path.Combine(Directory.GetCurrentDirectory(), "ExampleFile", "ExampleTours.json");
 	}
 
     public override void Execute()
     {
+		Testje20230326_001();
         //Testje20230320_001();
         //Testje20230320_002();
         //Testje20230320_003();
-        Testje20230320_004();
+        //Testje20230320_004();
+        //Testje20230320_005();
         //KevinDing();
         //EvenSchrijven();
         //KorteTest();
         //JsonPrutsen();
         //Testje20230318();
     }
 
+	private void Testje20230326_001()
+	{
+		var showTours = new ShowToursController();
+		showTours.Execute();
```

Comment:
In principe wil je "Execute" overlaten aan Program en dan hier alleen maar `NextController = showTours;`